python_modules/dagster/dagster/_core/definitions/freshness_checks/utils.py

```diff
@@ -20,7 +20,8 @@
 # Top-level metadata keys
 LAST_UPDATED_TIMESTAMP_METADATA_KEY = "dagster/last_updated_timestamp"
 FRESHNESS_PARAMS_METADATA_KEY = "dagster/freshness_params"
-OVERDUE_DEADLINE_TIMESTAMP_METADATA_KEY = "dagster/overdue_deadline_timestamp"
+EXPECTED_BY_TIMESTAMP_METADATA_KEY = "dagster/expected_by_timestamp"
```

Comment:
Can we include a comment here explaining what this metadata entry means? When we move to `FreshnessMetadataSet`, it will become the docstring on that parameter.

Reply:
Sounds good.